README.md

```diff
@@ -251,6 +251,7 @@ to execute each step, without further knowledge. Ask a developer who knows more.
 - Add country in IBF-pipeline settings.py with model = 'rainfall'
 - Save geoserver output as rainfall_extent_<leadTime>_<countrYCode>
 
+[Known Issues](https://github.com/rodekruis/IBF-system/issues/826).
```

Comment:
@SwatiTh this link is referring to the specific issue. This should link to the generic issue-section instead. But to be honest, if there are no relevant issues there at the moment yet, then we don't have to add anything for now.

Let me ask the overall check-question for this item again: do you feel confident that the next developer who joins the project, can set up the system from the current README by him/herself? (Even though you made no changes to it in the end?)